composer.json

```diff
@@ -34,5 +34,7 @@
         "psr-4": {
             "SimpleBus\\JMSSerializerBridge\\Tests\\": "tests"
         }
-    }
+    },
+    "minimum-stability": "dev",
```

Comment:
I don't think we should include this.

Reply:
AFAIK it should have no impact.

Reply:
It is a root-only node and should not have any effect on the requirement of the package.
https://getcomposer.org/doc/04-schema.md#minimum-stability

But why do we need this? I understand that symfony 4 is still in development at the moment but should we fix it like this or make it a `^4.0@dev` or wait until the end of the when so that symfony 4 is released?

Reply:
This usually has no effect in combination with "prefer-stable: true" for packages that have a stable version release for a version, but it is useful to check against versions not yet released and get prepared ahead of time.